recipes/fluidsynth/all/conanfile.py

```diff
@@ -130,22 +131,28 @@ def requirements(self):
     def source(self):
         tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
 
-    def _configure_cmake(self):
-        if self._cmake:
-            return self._cmake
-        self._cmake = CMake(self)
-        self._cmake.definitions["enable-debug"] = self.settings.build_type
-        self._cmake.definitions["enable-tests"] = False
-        self._cmake.definitions["LIB_INSTALL_DIR"] = "lib"  # https://github.com/FluidSynth/fluidsynth/issues/476
-        self._cmake.definitions["FRAMEWORK_INSTALL_DIR"] = os.path.join(self.package_folder, "Frameworks")
-        self._cmake.definitions["CMAKE_TOOLCHAIN_FILE"] = os.path.join(self.install_folder, "conan_toolchain.cmake")
+    def generate(self):
```

Comment:
I have never used this so far.

Should `pkg_config` also be part of `generate()`? Or is the split between the `generators` attribute and the `generate` method okay?

Reply:
actually there seem to be a bug in PkgConfigDeps generator, because build fails with `KeyError: {'ref': pcre/unknown@unknown/unknown, 'build': False, 'direct': True}`. I reverted the change